acceptance-tests/src/test/resources/features/site/search.feature

```diff
@@ -217,15 +217,3 @@ Feature: Basic search
         When I search for "SynonymTest"
         Then I should see 1 search result
         And I can click on link "synonym test"
-
```

Comment:
The test data used here does not contain a parent series, therefore information type is not inherited or available within these publications